tavern_server/src/db_tables.sql

```diff
@@ -0,0 +1,676 @@
+--enum types
+CREATE TYPE Gender AS ENUM (
+	'Male', 
+	'Female', 
+	'Other'
+);
+
+CREATE TYPE Size AS ENUM (
+	'Fine', 
+	'Diminutive', 
+	'Tiny', 
+	'Small', 
+	'Medium', 
+	'Large', 
+	'Huge', 
+	'Gargantuan', 
+	'Colossal'
+);
+
+CREATE TYPE Alignment AS ENUM (
+	'Lawful Good', 
+	'Lawful Neutral', 
+	'Lawful Evil', 
+	'Neutral Good', 
+	'True Neutral', 
+	'Neutral Evil', 
+	'Chaotic Good', 
+	'Chaotic Neutral', 
+	'Chaotic Evil'
+);
+
+CREATE TYPE Attribute AS ENUM (
+	'Strength', 
+	'Dexterity', 
+	'Constitution', 
+	'Intelligence', 
+	'Wisdom', 
+	'Charisma'
+);
+
+CREATE TYPE Skill AS ENUM (
+	'Acrobatics', 
+	'Appraise', 
+	'Bluff', 
+	'Climb', 
+	'Craft', 
+	'Diplomacy', 
+	'Disable Device', 
+	'Disguise', 
+	'Escape Artist', 
+	'Fly', 
+	'Handle Animal', 
+	'Heal', 
+	'Intimidate', 
+	'Knowledge (arcana)', 
+	'Knowledge (dungeoneering)', 
+	'Knowledge (engineering)', 
+	'Knowledge (geography)', 
+	'Knowledge (history)', 
+	'Knowledge (local)', 
+	'Knowledge (nobility)', 
+	'Knowledge (planes)', 
+	'Knowledge (religion)', 
+	'Linguistics', 
+	'Perception', 
+	'Perform', 
+	'Profession', 
+	'Ride', 
+	'Sense Motive', 
+	'Sleight of Hand', 
+	'Spellcraft', 
+	'Stealth', 
+	'Survival', 
+	'Swim', 
+	'Use Magic Device'
+);
+
+CREATE TYPE SaveThrow AS ENUM (
+	'Fortitude', 
+	'Reflex',
+	'Will'
+);
+
+CREATE TYPE CharacterStat AS ENUM (
+	'Name',
+	'Race',
+	'Size',
+	'Height',
+	'Weight',
+	'Age',
+	'Gender',
+	'Alignment',
+	'Deity',
+	'Languages',
+	'Appearance'
+);
+
+CREATE TYPE CombatStat AS ENUM (
+	'Melee Attack Bonus',
+	'Ranged Attack Bonus',
+	'CMB',
+	'CMD',
+	'Armor Class',
+	'Touch AC',
+	'Flat-Footed AC',
+	'Initiative Bonus',
+	'Damage Reduction',
+	'Spell Resistance',
+	'Speed',
+	'Fortititude Save',
+	'Reflex Save',
+	'Will Save'
+);
+
+CREATE TYPE RaceType AS ENUM (
```

Comment:
What about racial subtypes, e.g. the many subtypes of humanoid?

Reply:
I noticed the subtypes, but chose to omit them at first. I can add a RaceSubtype ENUM for the various racial subtypes - admittedly, I assumed they wouldn't be very important, but the SRD shows that subtypes matter a bit more than merely flavor and cosmetic.